index.html

```diff
@@ -1104,7 +1104,12 @@
 recognized. Note that sites can do harm even if they can't be completely certain
 that visits come from the same person, so [=user agents=] should also take steps
 to prevent such probabilistic recognition. The [[[Privacy-Threat]]] discusses
-the tradeoffs involved.
+the tradeoffs involved. Note that if a [=user agent=] blocks [=recognition=]
+between different sites but doesn't notice when a single site contains multiple
+[=contexts=], this can encourage [=actors=] to bundle multiple contexts into a
+single site and privilege the [=actors=] who can do so. These are both harmful,
+but that harm is likely less than the harm from allowing [=cross-site
+recognition=].
```

Comment:
The change is generally okay but I feel like the conclusion is a little weak.

Maybe replace that last sentence with: `The fact that [=user agents=] can't mitigate all issues does not entail that they should mitigate none. On the contrary, mitigating an important part of the problem makes it easier for other approaches (eg. regulatory) to focus on what technical means cannot address.` Feel free to change the words if you like the change, of course, the point is to convey that this is an "and" approach, and that if regulators don't have to worry about cookies they could target more interesting things.

Reply:
That seems reasonable. How's my new text? I reformatted into bullets to try to keep the section manageable.

Reply:
LGTM!